search_service/models/user.py

```diff
@@ -33,37 +22,23 @@ def get_id(self) -> str:
     @classmethod
     def get_attrs(cls) -> Set:
         return {
-            'name',
+            'full_name',
```

Comment:
@friendtocephalopods @jinhyukchang on our end its looking like the search service is not returning this value when we search for users. It looks like `_get_search_result` relies on the attributes having the same name as what is in the elasticsearch payload and it looks like the elasticsearch payload is using `name` instead of `full_name`. 

Is this speculation correct? If so how can we eventually fix this, do we need to change the elasticsearch attribute or the schema? Is there a quick fix that can be made?

Reply:
oh no! I think this is coming from a mismatch in databuilder's ESDocument class: https://github.com/lyft/amundsendatabuilder/pull/188/commits/d3382ee2d3de92f239b2a07da41819067c820853

One of the reasons we prefer APIs to the ETL approach in databuilder is because there's less shared, coupled config between ingestion and the services ;)

I think in this case you'd have to rerun databuilder with that commit and things should work.

Reply:
Good point @ttannis @friendtocephalopods 
I think we can also utilize amundsen-common between service and databuilder to narrow the gap.

Related issue is opened [here](https://github.com/lyft/amundsen/issues/296) and let's address this separately as it's already massive and we just finished testing with current changes.